pkg/kubelet/secret/secret_manager.go

```diff
@@ -226,6 +224,11 @@ func getSecretNames(pod *v1.Pod) sets.String {
 			}
 		}
```

Comment:
What is using EnvFrom ? From what I found in the code, volume secret plugin is not using them...